### ISSUE TITLE
[BUG] Crashes when opening 'Categories' menu

### ISSUE THREAD

**Issue by katacarbix** [otakuhqz/Omega#77]
## Description

When I try to open the "Categories" menu under "Drawer" in the settings, Omega launcher crashes.

## Expected Behavior

I would expect it to open the menu like it did just a few days ago when I was setting it up.

## Actual Behavior

Omega launcher crashes.

## Steps to Reproduce

1. Long press a blank area on the home screen and tap "Home settings".
2. Tap "Drawer".
3. Tap "Categories".
4. A screen showing a ladybug and "An unexpected error occurred"  will appear.

## Device and OS

* Device: Google Pixel 3a
* Android version: 11 (build RQ2A.210505.002)
* Launcher version: v0.7.1 (build 118)
* Rom: Stock Pixel rom

## Logcat

```
Build version: 0.7.1 
Build date: 1979-11-30 00:00:00 
Current date: 2021-05-13 01:28:09 
Device: Google Pixel 3a 
 
Stack trace:  
java.lang.NullPointerException: Trying to create invalid component key: mainline_train_primary:301801800/#UserHandle{0}
	at com.android.launcher3.Utilities.makeComponentKey(Utilities.java:944)
	at com.saggitt.omega.groups.DrawerTabs$PredefinedTab.getSummary(DrawerTabs.kt:106)
	at com.saggitt.omega.groups.ui.AppGroupsAdapter$GroupHolder.bind(AppGroupsAdapter.kt:200)
	at com.saggitt.omega.groups.DrawerTabsAdapter$TabHolder.bind(DrawerTabsAdapter.kt:66)
	at com.saggitt.omega.groups.ui.AppGroupsAdapter.onBindViewHolder(AppGroupsAdapter.kt:76)
	at com.saggitt.omega.groups.ui.AppGroupsAdapter.onBindViewHolder(AppGroupsAdapter.kt:36)
	at androidx.recyclerview.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:7065)
	at androidx.recyclerview.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:7107)
	at androidx.recyclerview.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:6012)
	at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6279)
	at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6118)
	at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6114)
	at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2303)
	at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1627)
	at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1587)
	at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:665)
	at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:4134)
	at androidx.recyclerview.widget.RecyclerView.onMeasure(RecyclerView.java:3540)
	at android.view.View.measure(View.java:25466)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1552)
	at android.widget.LinearLayout.measureVertical(LinearLayout.java:842)
	at android.widget.LinearLayout.onMeasure(LinearLayout.java:721)
	at android.view.View.measure(View.java:25466)
	at androidx.core.widget.NestedScrollView.measureChildWithMargins(NestedScrollView.java:1599)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at androidx.core.widget.NestedScrollView.onMeasure(NestedScrollView.java:585)
	at android.view.View.measure(View.java:25466)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at android.view.View.measure(View.java:25466)
	at android.widget.LinearLayout.measureVertical(LinearLayout.java:995)
	at android.widget.LinearLayout.onMeasure(LinearLayout.java:721)
	at android.view.View.measure(View.java:25466)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at android.view.View.measure(View.java:25466)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at android.view.View.measure(View.java:25466)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at android.view.View.measure(View.java:25466)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at android.view.View.measure(View.java:25466)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at androidx.appcompat.widget.ContentFrameLayout.onMeasure(ContentFrameLayout.java:146)
	at android.view.View.measure(View.java:25466)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1552)
	at android.widget.LinearLayout.measureVertical(LinearLayout.java:842)
	at android.widget.LinearLayout.onMeasure(LinearLayout.java:721)
	at android.view.View.measure(View.java:25466)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at android.view.View.measure(View.java:25466)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1552)
	at android.widget.LinearLayout.measureVertical(LinearLayout.java:842)
	at android.widget.LinearLayout.onMeasure(LinearLayout.java:721)
	at android.view.View.measure(View.java:25466)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6957)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at com.android.internal.policy.DecorView.onMeasure(DecorView.java:747)
	at android.view.View.measure(View.java:25466)
	at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:3402)
	at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:2246)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2504)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1948)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:8177)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:972)
	at android.view.Choreographer.doCallbacks(Choreographer.java:796)
	at android.view.Choreographer.doFrame(Choreographer.java:731)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:957)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7660)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```

**Comment by machiav3lli** [comment]
This is already fixed in the latest 0.8.0-test builds.

**Comment by katacarbix** [comment]
Great, thank you!
